src/renderer/components/+workloads-pods/pod-details-affinities.tsx

```diff
@@ -42,11 +46,12 @@ export class PodDetailsAffinities extends React.Component<Props> {
       <DrawerItem name="Affinities" className="PodDetailsAffinities">
         <DrawerParamToggler label={affinitiesNum}>
           <div className="ace-container">
-            <AceEditor
-              mode="yaml"
+            <MonacoEditor
+              options = {{ readOnly: true, ...UserStore.getInstance().getEditorOptions(EditorType.DETAILS)}}
+              className={cssNames( "MonacoEditor")}
+              theme={ThemeStore.getInstance().activeTheme.monacoTheme}
+              language="yaml"
```

Comment:
Please consider to create wrapper for `MonacoEditor` from `react-monaco-editor` in order to prevent passing duplicated props for every editor instance such as:
* className
* theme
* language

Reply:
I think that language should be always specified.

Reply:
100% of the time we use `yaml` so it can be just default value without passing it everywhere.